chapters/synchronization.txt

```diff
@@ -70,32 +70,37 @@ If an operation happens-before another operation, then the first operation
 must: complete before the second operation is initiated.
 More precisely:
 
-  * Let *A* and *B* be separate sets of operations.
-  * Let *S* be a synchronization command.
-  * Let *A~S~* and *B~S~* be the synchronization scopes of *S*.
-  * Let *A'* be the intersection of sets *A* and *A~S~*.
-  * Let *B'* be the intersection of sets *B* and *B~S~*.
-  * Submitting *A*, *S* and *B* for execution, in that order, will result in
-    execution dependency *E* between *A'* and *B'*.
-  * Execution dependency *E* guarantees that *A'* happens-before *B'*.
+  * Let *Ops~1~* and *Ops~2~* be separate sets of operations.
+  * Let *Sync* be a synchronization command.
+  * Let *Scope~1st~* and *Scope~2nd~* be the synchronization scopes of
+    *Sync*.
+  * Let *ScopedOps~1~* be the intersection of sets *Ops~1~* and
+    *Scope~1st~*.
+  * Let *ScopedOps~2~* be the intersection of sets *Ops~2~* and
+    *Scope~2nd~*.
+  * Submitting *Ops~1~*, *Sync* and *Ops~2~* for execution, in that order,
+    will result in execution dependency *ExeDep* between *ScopedOps~1~* and
+    *ScopedOps~2~*.
+  * Execution dependency *ExeDep* guarantees that *ScopedOps~1~*
+    happen-before *ScopedOps~2~*.
 
 [[synchronization-dependencies-chains]]
 An _execution dependency chain_ is a sequence of execution dependencies that
-form a happens-before relation between the first dependency's *A'* and the
-final dependency's *B'*.
+form a happens-before relation between the first dependency's *ScopedOps~1~*
+and the final dependency's *ScopedOps~2~*.
 For each consecutive pair of execution dependencies, a chain exists if the
-intersection of *B~S~* in the first dependency and *A~S~* in the second
-dependency is not an empty set.
+intersection of *Scope~2nd~* in the first dependency and *Scope~1st~* in the
+second dependency is not an empty set.
 The formation of a single execution dependency from an execution dependency
 chain can be described by substituting the following in the description of
 execution dependencies:
 
-  * Let *S* be a set of synchronization commands that generate an execution
-    dependency chain.
-  * Let *A~S~* be the first synchronization scope of the first command in
-    *S*.
-  * Let *B~S~* be the second synchronization scope of the last command in
-    *S*.
+  * Let *SyncChain* be a set of synchronization commands that generate an
```

Comment:
I don't think this is quite right - it is saying to substitute these terms in the language above, but there \*Sync* is referred to, not \*SyncChain*. It almost works except for the "Submitting \*Ops~1~*, \*Sync* and \*Ops~2~* for execution," clause. Using the same label for \*Sync* and \*SyncChain* would address this.

Reply:
Ok, sticking with the original "Sync" name for now.